charts/kubernetes-external-secrets/templates/rbac.yaml

```diff
@@ -30,27 +35,36 @@ rules:
     verbs: ["get", "update"]
 ---
 apiVersion: rbac.authorization.k8s.io/v1beta1
-kind: ClusterRoleBinding
+kind: {{ include "kubernetes-external-secrets.rbacBindingObject" . }}
```

Comment:
I could refactor this to `kind: {{ include "kubernetes-external-secrets.rbacObject" . }}Binding`, and get rid of the `kubernetes-external-secrets.rbacBindingObject` template, but I think this is cleaner.

Reply:
agree this seems cleaner